index.js

```diff
@@ -1,30 +1,37 @@
-module.exports = {
+
+import stylistic from "@stylistic/eslint-plugin-js";
+
+export default {
+	plugins: {
+		"@stylistic/js": stylistic
+	},
 	rules: {
-		"no-negated-in-lhs": "error",
+		"no-unsafe-negation": "error",
 		"no-cond-assign": [ "error", "except-parens" ],
-		curly: [ "error", "all" ],
-		"object-curly-spacing": [ "error", "always" ],
-		"computed-property-spacing": [ "error", "always" ],
-		"array-bracket-spacing": [ "error", "always" ],
-		eqeqeq: [ "error", "smart" ],
+		"curly": [ "error", "all" ],
```

Comment:
Why the change to quote the keys?

We should have a rule for that, BTW...

Reply:
I think my IDE did that. It's a rule in another project I was working on. I can change them back if you want.

Reply:
Yes, please avoid changes like that if they are not a result of a code style rule change.